presentation/index.html

```diff
@@ -1,371 +1,368 @@
+                        <li class='fragment'> One large group presentation per team </li>
+                    </ul>
+                </section>
+
+                <section>
+                    <h2> Questions? </h2>
+                </section>
+
+                <!-- Groups Present -->
+
+                <section>
+                    <h2> Getting started at RCOS </h2>
+                </section>
+
+                <section>
+                    <h2> Observatory (<a href='http://rcos.io'>rcos.io</a>) </h2>
+                    <p> RCOS's homebrew project management system </p>
+                    <img src='img/observatory.png'/>
+                </section>
+
+                <section>
+                    <h2> Slack (<a href='http://rcos.slack.com'>rcos.slack.com</a>) </h2>
+                    <p> RCOS's primary communication medium </p>
+                    <img src='img/slack.png'/>
+                </section>
+
+                <section>
+                    <h2>0-credit SIS course</h2>
+                    <h3>(SRN 97169 CSCI-4963-01) </h3>
+                    <p> To reserve our rooms and get a roster </p>
+                </section>
+
+                <section>
+                    <h2><a href='http://github.com'>Github</a></h2>
+                    <img src='img/github.png'/>
+                </section>
+
+                <section>
+                    <h2> Finding a project </h2>
+                    <a href='https://github.com/explore'>github.com/explore</a>
+                </section>
+
+                <section>
+                    <h2> Finding a team (optional) </h2>
+                    <ul>
+                        <li class='fragment'> Pitch Day </li>
+                        <li class='fragment'> Post on Slack </li>
+                        <li class='fragment'> Talk to your mentor </li>
+                    </ul>
+                </section>
+
+                <section>
+                    <h2> Start learning git (<a href='http://try.github.io'>try.github.io</a>) </h2>
+                    <img src='img/learngit.png'/>
+                </section>
+
+                <section>
+                    <h2> Getting Help </h2>
+                    <ul class='fragment'>
+                        <li class='fragment'> #helpdesk </li>
+                        <li class='fragment'> Slack </li>
+                        <li class='fragment'> Mentor </li>
+                        <li class='fragment'> Coordinator </li>
+                        <li class='fragment'> Faculty Member </li>
+                    </ul>
+                </section>
+
+                <section>
+                    <h2> How to be successful (and get an A!) </h2>
+                    <ul>
+                        <li class='fragment'> Commit often (shoot for once a week) </li>
+                        <li class='fragment'> Tell us what you're up to (blog ~3 times a semester) </li>
```

Comment:
what should we put in the place here? there is a wide range of credit levels from 1-4.